website/static/css/pages/profile-page.css

```diff
@@ -38,3 +38,37 @@ table.table.table-plain th, table.table.table-plain td {
 #social td:first-child {
     word-break: break-all;
 }
+
+.fa.toggle-icon {
+    position: absolute;
+    right: 0;
+    top: 50%;
+    transform: translate(0, -50%);
+    -webkit-transform: translate(0, -50%);
+    -ms-transform: translate(0, -50%);
+    display: inline-block;
+    margin: 0 15px 0 0;
+}
+
+/* Cards
+-------------------- */
+
+.institution {
```

Comment:
This looks brittle; what if we change the order of columns? Can you make this more generic. Maybe something along the lines of `.table-responsive td:first-child`?